internal/eventstore/v1/spooler/spooler.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/caos/zitadel/internal/view/repository"
 )
 
+const systemID = "system"
```

Comment:
I dont get why we have this fixed instance id on the spooler

Reply:
see above